### DISCUSSION TITLE
Github Education

### DISCUSSION THREAD

**Discussion by Vinay0180** [community/community#42102] [GitHub Education]
Hi,

I am a current student at UCI Paul Merage School of business pursuing my masters in business analytics. I have been trying to validate my student status to access the github student developer pack but I am not able to do so. No matter what proof I submit, it asks me for a dated student id. My student id does not have a date on it, I am ready to submit my unofficial transcripts, i-20, offer letter, whatever is needed.

I requested my university for a dated student id, but unfortunately they said thats the general standard for masters student. I request for your urgent help!

**Comment by samus-aran** [comment]
Hi @Vinay0180 👋 

We're happy you're here. A couple of other students have asked similar questions. Let me [link in their discussion ](https://github.com/orgs/community/discussions/42018)and hopefully the comments will help guide you to submit a successful application.
